Documentation/common_server_docs.py

```diff
@@ -131,6 +131,7 @@ def createJsDocumentation(path, origin, language):
 
 
 def createPyDocumentation(path, origin, language):
+    ignore_names = ('DemistoException', 'BaseClient')
```

Comment:
why not to use pyPrivateFuncs

Reply:
It's not clear what `pyPrivateFuncs` is for. will add comment

> why not to use pyPrivateFuncs
It's not clear what `pyPrivateFuncs` is for. Will add comment